generator/shared.go

```diff
@@ -1257,3 +1268,10 @@ func sharedValidationsFromSchema(v spec.Schema, isRequired bool) (sh sharedValid
 	}
 	return
 }
+
+func stringTernary(cond bool, ifTrue string, ifFalse string) string { // nolint: unparam
```

Comment:
the // nolint: hint is in my opinion unnecessary. You could just declare this has: (cond bool, ifTrue, ifFalse string) string